pymc/tests/test_mixture.py

```diff
@@ -721,6 +722,31 @@ def test_broadcast_components(self, comp_dists, expected_shape):
             assert isinstance(comp_dist.owner.op, RandomVariable)
             assert tuple(comp_dist.shape.eval()) == expected_shape
 
+    @pytest.mark.parametrize(
+        "comp_dists",
+        [
+            (
+                Categorical.dist(np.tile(1 / 3, 3)),
+                Normal.dist(np.ones(3), 3),
+            ),
+            (
+                Binomial.dist(n=10, p=0.5),
+                Normal.dist(),
+            ),
+            (
+                Categorical.dist(np.broadcast_to(1 / 3, (5, 2, 3))),
+                MvNormal.dist(np.ones(3), np.eye(3), shape=(5, 2)),
+            ),
+        ],
+    )
+    def test_preventing_mixing_cont_and_discrete(self, comp_dists):
```

Comment:
One test is probably enough here

Reply:
Should I just include it in the function rather than use `pytest.mark.parametrize`?

Reply:
Yup